app/controllers/admin/paypal_preferences_controller.rb

```diff
@@ -38,10 +38,16 @@ def index
     currency = @current_community.default_currency
     minimum_commission = paypal_minimum_commissions_api.get(currency)
 
+    tx_settings =
+      Maybe(tx_settings_api.get(community_id: @current_community.id, payment_gateway: :paypal, payment_process: :preauthorize))
```

Comment:
This same code is used in paypal_helper. Should the paypal helper contain a method like
```ruby
def get_settings
  Maybe(TxApi.settings.get_active(community_id: community_id))
      .select { |result| result[:success] }
      .map { |result| result[:data] }
end
```